scripts/check_frontend_test_coverage.py

```diff
@@ -356,6 +356,7 @@
     'tutor-card.directive.ts',
     'unicode-string-editor.directive.ts',
     'unit-test-utils.ts', # Please don't try to cover this file.
+    'upload-activity-modal.component.ts',
```

Comment:
No adding to this file please...

Reply:
Please refer this comment https://github.com/oppia/oppia/pull/12290/files#r602721337